src/components/Comment/styles.module.scss

```diff
@@ -0,0 +1,38 @@
+@import '../../scss/variables.scss';
+
+$padding: 10px;
+
+.container {
+  padding: 10px;
+  width: 100%;
+}
+
+.comment,
+.form {
+  background-color: $light-gray;
+  border: 1px solid darken($light-gray, 10);
+  padding: $padding;
+}
+
+.textarea {
+  display: block;
+  margin-bottom: $padding;
+  width: 100%;
+}
+
+.comment {
+  align-items: center;
+  display: flex;
+  // FontAwesome icon is 32px.
+  min-height: 32px + $padding + $padding;
+  // We add a padding-left to position the content after the icon.
+  padding-left: 32px + $padding + $padding;
+  position: relative;
+  width: 100%;
+}
+
+.commentIcon {
+  position: absolute;
```

Comment:
Why do you need absolute positioning for this? Were you trying to keep the icon at the top for a multi-line comment? If so, flex or grid should work for that.

Reply:
Exactly, it is to keep the comment in the top-left corner.